docs/prms/allpairs.rst

```diff
@@ -8,3 +8,17 @@ with respect to a cartesian product.
 
 All Pairs does not take any arguments. Its source code is at https://github.com/Reed-CompBio/all-pairs-shortest-paths
 licensed under MIT.
+
+Dataset Usage
+-------------
+
+All Pairs Shortest Paths uses ``sources``, ``targets``, and edge weights.
+All Pairs Shortest Paths accepts directed graphs.
```

Comment:
```suggestion
All Pairs Shortest Paths input requires the interactome to be directed.
```

Reply:
How about

```suggestion
All Pairs Shortest Paths considers graph directionality.
```

?

Reply:
The wording considers graph directionality doesn't make sense. We need to be more explicit. We can try All Pairs Shortest Paths input requires the interactome edges to be directed.

Reply:
Ok, I think we need something that encapsultes "All Pairs Shortest Paths input requires the interactome edges to all be directed or all be undirected. (APSP doesn't allow for interactomes that ar e a mix of undirected and directed edges)" after remembering what APSP does with the input interactomes.

Reply:
> The word directionality refers to whether edges are directed or undirected, so I’m not sure what you mean by “considers directionality.” When does an algorithm not consider directionality?

I see. I'll be more specific here when I do another pass.